client/index.js

```diff
@@ -195,6 +195,38 @@ function body(name) {
     li.appendChild(textSpan);
     history.insertBefore(li, history.firstChild);
   };
+
+  const subjectElement = document.querySelector('#subject');
+  eventHandlerMap['setSubject'] = (data) => {
+    subjectElement.innerText = data.subject;
+  };
+
+  const timeElement = document.querySelector('#time');
+  var countDownTimer = null;
+  var countDownEndTime = null;
+  eventHandlerMap['startTimer'] = (data) => {
+    if (countDownTimer !== null) {
+      clearInterval(countDownTimer);
+      countDownTimer = null;
+    }
+
+    countDownEndTime = (new Date()).getTime() + data.timeLimit;
```

Comment:
nitpick: () around `new Date()` is not necessary.
`Date.now()` does the same.

Reply:
Fixed :P